docs/src/main/asciidoc/openshift-s2i-guide.adoc

```diff
@@ -0,0 +1,51 @@
+include::./attributes.adoc[]
+= {project-name} - Deploying on OpenShift with S2I
+
+This guide covers:
+
+* The deployment of the application to OpenShift using S2I to build
+
+== Prerequisites
+
+For this guide you need:
+
+* roughly 10 minutes
+* having access to an OpenShift cluster. Minishift is a valid option.
+
+== Solution
+
+We recommend to follow the instructions in the next sections and build the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `getting-started-openshift-s2i` directory.
+
+== Deploying the application in OpenShift
+
+In this section, we are going to leverage the S2I build mechanism of OpenShift.
+We use a Java S2I Builder, and therefore do not need a `Dockerfile` in this approach.
+You do not need to locally clone the Git repository, as it will be directly built since OpenShift.
+We are going to create an OpenShift `build` executing it:
+
+[source,shell]
+----
+# To build the image on OpenShift
+oc new-app registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started-openshift-s2i --name=quarkus-quickstart
```

Comment:
The `context-dir` here will have to be changed, following the decision in https://github.com/quarkusio/quarkus-quickstarts/pull/81 ...